admin_manual/appliance/howto-update-owncloud.rst

```diff
@@ -162,6 +163,14 @@ This is because the earlier versions of ownCloud are installed with a different
 More specifically, the versions of the ownCloud app, prior to 10, have a version suffix in the name. 
 For example the ownCloud 8.2 app is named ``owncloud82``.
 
+For ownCloud 8.2 users: during the ownCloud App upgrade, user files will be moved to the new Docker data directory, ``/var/lib/univention-appcenter/apps/owncloud/data/files``.  Essentially, the following the command will be executed:
```

Comment:
One too many "the" in the sentence.

s/Essentially, the following the command/Essentially, the following command/g

Reply:
thanks